content/guides/accessibility/color-considerations.mdx

```diff
@@ -0,0 +1,332 @@
+
+#### Forced colors mode
+
+This **is not** provided by GitHub.
+
+Forced color mode is another media query. Unlike high contrast mode, forced colors mode automatically adapts **all** operating system content. This includes **all** web content.
+
+A forced color mode will update all GitHub content's visual presentation regardless of what theme a user has selected. It uses a suite of heuristics to determine content meaning and then assign a dynamic color keyword value to it.
+
+Forced color mode content presentation **should not** be overridden to satisfy aesthetic desires. Instead, it should only be lightly adjusted to ensure maximum legibility and accuracy of applied color keywords.
+
+## Inputs and buttons
+
+**Different kinds of UI have their contrast compliance determined in different ways**. This is because different parts of them are visually significant for understanding purpose.
+
+The difference between **inputs** and **buttons** may be confusing. Understanding the difference between the two types of UI helps to understand why they are evaluated differently for color contrast purposes.
+
+At a high-level:
+
+- Buttons **perform a predetermined action** when pressed. Examples of this are submitting form input, triggering a dialog, or saving user-entered content.
+- Inputs **collect information for eventual submission**. Examples of this are single and multiline text-entry areas, checkbox selections/radio choices, and comboboxes.
+
+Inputs and buttons are combined to form data collection and transmission experiences such as a contact form or a Pull Request submission. 
+
+### Inputs
+
+Being able to perceive the boundaries of an input is vital. Meeting WCAG color contrast requirements helps to ensure the widest possible range of users can perform the data entry required to operate GitHub successfully.
+
+#### Input considerations
+
+Broadly-speaking, there are four overall considerations for input contrast:
+
+1. Text,
+2. Borders, 
+3. Selected state, and
+4. Validation state, and
+
+Combined, meeting contrast requirements allow a user to understand:
+
+1. What the input is for,
+2. What area of the input a user can interact with, 
+3. What information a user has entered into the input, and
+4. What state the input is in.
+
+If a computed color value in any of the four considerations is not compliant, it may endanger one or more portions of user understanding and operation of the input.
+
+#### Input text contrast
+
+Text portions of an input **must** meet a contrast ratio of `4.5:1` for "normal text", or `3:1` for "large text". This applies to both an input's visible label and its user-provided text content.
+
+<img src={InputText} alt="A wireframe illustration of a text input. Its label reads, 'Name.' It's user-provided text content reads, 'Mona Lisa.' A label pointing to the label reads, 'Communicates purpose.' A label pointing to the user provided content reads, 'Communicates user choice.'" />
```

Comment:
To me, it makes more sense to have this image before the input text contrast section so that it's in the input consideration section